apps/staking/components/StakedNodeCard.tsx

```diff
@@ -743,7 +742,7 @@ const StakedNodeCard = forwardRef<
     const deregistrationUnlockTime = useRelativeTime(deregistrationUnlockDate, { addSuffix: true });
     const liquidationTime = useRelativeTime(liquidationDate, { addSuffix: true });
 
-    const isSoloNode = contributors.length === 1;
+    const isSoloNode = contributors.length === 1 && !node.contract;
```

Comment:
Can you elaborate on how a solo node does not have a contract?

Reply:
because its a solo node. Multi-nodes are the onces with a contract

Reply:
Discussed and resolved